catalog/app/containers/Bucket/requests/package.ts

```diff
@@ -183,28 +180,8 @@ export function useCreatePackage() {
   )
 }
 
-const updatePackage = (
-  req: ApiRequest,
-  { contents, message, meta, target, workflow }: UpdatePackageParams,
-  schema: JsonSchema, // TODO: should be already inside workflow
-) =>
-  uploadManifest(req, ENDPOINT_UPDATE, {
-    name: target.name,
-    registry: `s3://${target.bucket}`,
-    message,
-    contents,
-    meta: getMetaValue(meta, schema),
-    workflow: getWorkflowApiParam(workflow.slug),
-  })
-
-export function useUpdatePackage() {
-  const req: ApiRequest = APIConnector.use()
-  return React.useCallback(
-    (params: UpdatePackageParams, schema: JsonSchema) =>
-      updatePackage(req, params, schema),
-    [req],
-  )
-}
+// backwards compatibity, can be removed soon
+export { useCreatePackage as useUpdatePackage }
```

Comment:
You can remove it already

Reply:
yep, i kept it to prevent that single commit from breaking the code